GUI/Types/Renderer/Shaders/simple.frag

```diff
@@ -62,8 +62,8 @@ void main()
 #else
     //Calculate half-lambert lighting
     float illumination = dot(worldNormal, lightDirection);
-    illumination = illumination * 0.5 + 0.5;
-    illumination = illumination * illumination;
+    //illumination = illumination * 0.5 + 0.5;
+    //illumination = illumination * illumination;
```

Comment:
This should be reverted, this formula is the half-lambert equation Valve is known to use.

Reply:
I found mentions only about Source 1. Source 2 should have a more physically accurate lighting model.

Reply:
Unless we find explicit references for this we assume Source 2 is doing the same as Source 1.

Reply:
http://media.steampowered.com/apps/valve/2015/Alex_Vlachos_Advanced_VR_Rendering_GDC2015.pdf
page 42

Reply:
Very interesting, in that case can you remove the old commented code?

Reply:
Sure